File: C_Programing/Competitive/Problems_Solving/ASSIGNMENT_6/Q5.c
//WAP which accept no from user and display  its table in reverse order 



#include<stdio.h>


    void TableRev(int iNo)
{

    if(iNo < 0)
    {
        iNo = -iNo;    // Negative number positive करा
    }

    int iCnt = 0;
   

    for(iCnt = 10; iCnt >=1; iCnt--) //TAble logic
    {
        printf("%d\n" ,iNo * iCnt);
    }
   
}


int main()
{
    int iValue = 0;

    printf("Enter number   : ");
    scanf("%d",&iValue);

  TableRev(iValue);


    return 0;
}

File: java_Programing/Competitive/Problems_Solving/ASSIGNMENT_18/Q5.java
// WAP to print the multiplication table of a number

import java.util.Scanner;

class Logy
{
    void Table(int NO)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= 10; iCnt++)
        {
            System.out.println(NO + " * " + iCnt + " = " + (NO * iCnt));
        }
    }
}

public class Q5
{
    public static void main(String A[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter Number : ");
        iValue = sobj.nextInt();

        Logy lobj = new Logy();

        lobj.Table(iValue);
    }
}